domains/Nodirbek.json

```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "Piratebay298",
+        "email": "piratebay298@gmail.com"
+      },
+
+      "record": {
+        "CNAME": "https://github.com/Piratebay298/"
```

Comment:
CNAME cannot contain `https://` or `/`. I would recommend changing this to a `URL` record.